### ISSUE TITLE
Master Test Plan

### ISSUE THREAD

**Issue by vchelaru** [vchelaru/Gum#390]
- [ ] Polygon (mainly points)
- [ ] Behaviors
- [ ] Texture coordinates/wrapping
- [ ] Inheritance
- [ ] Animations
- [ ] Code generation
    - [ ] Fully generated
    - [ ] Loaded from .gumx
- [ ] AnimationChains (.achx)
- [ ] Text wrapping/bbcode/etc
- [ ] Fonts
- [ ] Variable References
- [ ] Skia (no monogame)

**Comment by kaltinril** [comment]
Marking Polygon complete, as I found no other issues that are not in it's own ticket

**Comment by kaltinril** [comment]
Behaviors are done, minus the few issues remaining, clicking the checkmark for behaviors

**Comment by kaltinril** [comment]
All items for Texture Coordinates/wrapping are complete or issues created for them or PRs in gitbook, clicke the checkmark on this section.

**Comment by kaltinril** [comment]
Exposed Variables are complete.

**Comment by kaltinril** [comment]
Finished reviewing Inheritance docs and testing.

**Comment by kaltinril** [comment]
@vchelaru 
Documentation needs code review for changes I've submitted 3-4 changes/PRs.

Also, the Animations need new images on the 3rd page.

Also, the 4th page (4 - Combining Multiple Categories) seems maybe it's missing information?  It just feels very short to be an entire page.

**Comment by vchelaru** [comment]
I believe all animation issues are resolved, and I've accepted the docs changes.

**Comment by kaltinril** [comment]
I feel 90-95% confident on animations.  Added a issue for the "Named Event" button that is not described anywhere.

I get an odd ever clicking around while the animations are running but it's impossible to reproduce.

**Comment by kaltinril** [comment]
@vchelaru  tired of testing XML, so here is what I have.

The excel file is the testing, the word is images or more explanation if I felt a specific TestID + Item was needed.

[corruption_testing_results.xlsx](https://github.com/user-attachments/files/18553311/corruption_testing_results.xlsx)
[xml_corruption_testing_results.docx](https://github.com/user-attachments/files/18553310/xml_corruption_testing_results.docx)

**Comment by kaltinril** [comment]
@vchelaru  for animation chains, I can't really "Test" it because it's not part of gum.  Did you want to give me an overview of them so I can learn them, or?  I reviewed the docxs and they work fine for an overview.

I didn't review the LINK that goes to the FRB docs.  That felt like an external thing.  

How do you want me to proceed?

**Comment by kaltinril** [comment]
@vchelaru  I do not understand the BBCode.  I tried it in monogame and it didn't appear to work.  After reviewing the Known Issues page, this appears to be one of them.

**Comment by vchelaru** [comment]
@kaltinril  for animations, yes I'd be happy to give a review of how they work. If we can't align on schedule, then feel free to look over the FRB Docs. You don't necessarily need to give feedback on the FRB docs, though. In fact, the FRB docs for the AnimationEditor reference an AnimationEditor tool which is still used heavily, but it is no longer being maintained because of the difficulty of working with it. I can go into details if you are interested.

Lately I've been talking to @vicdotexe  who has been working on a replacement for the AnimationEditor, and I'd rather not touch the FRB docs until that's ready to take over.

As far as bbcode - yes it is not currently supported in MonoGame; however, I think I should add it because it's actually not a lot of work. I'll create a card for it.

**Comment by kaltinril** [comment]
@vchelaru  i think this is as done as it's going to get unless you see further work we didn't cross off?

**Comment by vchelaru** [comment]
I agree, we've moved beyond this now and are getting plenty of issues from the community, from internal development, and your work with videos etc. Closing it out. Thanks!
